run_unit_tests.sh

```diff
@@ -61,21 +65,40 @@ if [ "$RUN_TESTS" == "1" ] ; then
             compress_logs && exit 1; } && \
             generate_coverage_report;
     fi
-    bazel build --jobs=$JOBS ${debug_bazel_flags} //src:ovms_test 
+    bazel build --jobs=$JOBS ${debug_bazel_flags} //src:ovms_test || exit 1
     set +x
     echo "Executing unit tests"
     failed=0
     # Tests starting python interpreter should be executed separately for Python 3.12 due to issues with multiple reinitialization of the interpreter
     for i in `./bazel-bin/src/ovms_test --gtest_list_tests --gtest_filter="-LLMChatTemplateTest.*:LLMOptionsHttpTest.*" | grep -vE '^ ' | cut -d. -f1` ; do
-        ./bazel-bin/src/ovms_test --gtest_filter="$i.*" > tmp.log 2>&1 || ( failed=1 ; echo $i ; cat tmp.log ) 
+        if OPENVINO_TOKENIZERS_PATH_GENAI=/opt/intel/openvino/runtime/lib/intel64/libopenvino_tokenizers.so  ./bazel-bin/src/ovms_test --gtest_filter="$i.*" > tmp.log 2>&1 ; then
```

Comment:
This value is used twice:
/opt/intel/openvino/runtime/lib/intel64/libopenvino_tokenizers.so
Perhaps we could move it to the variable?

Reply:
I was trying it but it wasn't effective for the test binary